src/components/Widget.tsx

```diff
@@ -154,7 +154,10 @@ export function TestableWidget(props: PropsWithChildren<TestableWidgetProps>) {
         props.jsonRpcUrlMap[supportedChainId as number] = fallbackRpc
       }
     }
-    return props.jsonRpcUrlMap
+    return Object.entries(props.jsonRpcUrlMap).reduce(
+      (urlMap, [id, urls]) => ({ ...urlMap, [id]: Array.isArray(urls) ? urls : [urls] }),
```

Comment:
is the `.isArray` call necessary here with the given jsonRpcUrlMap type?

Reply:
This is converting the value from `string[] | string` to `string[]`, so `.isArray` is the easiest way to disambiguate here.